scripts/docker_build/sof_builder/Dockerfile

```diff
@@ -59,8 +59,8 @@ ENV HOME /home/sof
 
 # Use ToT alsa utils for the latest topology patches.
 RUN mkdir -p /home/sof/work/alsa && cd /home/sof/work/alsa && \
-git clone $CLONE_DEFAULTS https://github.com/thesofproject/alsa-lib.git && \
-git clone $CLONE_DEFAULTS https://github.com/thesofproject/alsa-utils.git && \
+git clone $CLONE_DEFAULTS https://github.com/alsa-project/alsa-lib.git && \
```

Comment:
I would hold off here as there are some alsa-lib patches being worked on for topology 2.0. Best to just update our local copies of alsa-lib etc in the mean time.